src/client/containers/ChannelMessagesView/ChannelMessagesView.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import MessageList from '../../components/MessageList/MessageList';
+import PropTypes from 'prop-types';
+
+function ChannelMessagesView(props) {
+  return (
+    <ul className="container">
```

Comment:
If you are not styling it, then just rename the class name to `channel-messages`. The class name `container` is a bit too generic.